src/lib.rs

```diff
@@ -65,4 +65,5 @@ pub static MOTIVATIONS: [&'static str; 66] = [
   , "Be the person you'd like to meet one day"
 	, "RailsClub 2016 was rusted <3"
   , "You're good. Keep going and you will discover what apostrophes do in Rust"
+  , "Of Course We Sill Love You"
```

Comment:
it's a mistype, isn't it?

Reply:
what do you think is a mistype, @igorlimansky ? i'm not sure what you mean.

Reply:
Maybe "I" vs "we". I think "we" makes more sense for this message.

Reply:
@listochkin @ashleygwilliams hey, sorry for a pause :)
I mean, 'Sill' should be 'Still', shouldn't it?